### ISSUE TITLE
`.message` is not defined

### ISSUE THREAD

**Issue by andreyvital** [bjyoungblood/es6-error#4]
When used with babel, `e.message` returns nothing.

```
➜ ~ % node -v
v4.0.0
```

``` JS
import ExError from 'es6-error'

export default class UserNotFound extends ExError {
}

try {
  throw new UserNotFound(':)')
} catch (e) {
  console.log(e.message)
  console.log(typeof e.message)
  console.log(e.message.length)
}
```

:arrow_right: 

```

string
0
```

**Comment by tchak** [comment]
There is a pull request #3 to fix this

**Comment by zallek** [comment]
:+1:

**Comment by bjyoungblood** [comment]
Merged #3
